frontend/src/views/Game.tsx

```diff
@@ -1,10 +1,28 @@
 import Editor from '@monaco-editor/react';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { Room } from '../api/Room';
+
+type LocationState = {
+  room: Room,
+}
 
 function GamePage() {
+  const location = useLocation<LocationState>();
+  const [room, setRoom] = useState<Room | null>(null);
+
+  // Called every time location changes
+  useEffect(() => {
+    if (location && location.state && location.state.room) {
+      setRoom(location.state.room);
+    }
+  }, [location]);
+
   return (
     <div>
-      Game page
+      Room:
+      {' '}
+      {room ? room.roomId : 'No room joined'}
```

Comment:
When this is combined with @helenli8181's PR #22 in `master`, this will look weird; imo, whoever's PR is merged second should try to account for this. This likely means @helenli8181 will make another change in her PR to account for making this an OK design.